curvefs/src/metaserver/heartbeat.cpp

```diff
@@ -221,10 +220,9 @@ void Heartbeat::BuildCopysetInfo(curvefs::mds::heartbeat::CopySetInfo *info,
 bool Heartbeat::GetMetaserverSpaceStatus(MetaServerSpaceStatus* status,
                                          uint64_t ncopysets) {
     StorageStatistics statistics;
-    auto kvStorage = GetStorageInstance();
-    bool succ = kvStorage->GetStatistics(&statistics);
+    bool succ = GetResourceStatistic(&statistics);
     if (!succ) {
-        LOG(ERROR) << "Get storage statistics failed";
+        LOG(WARNING) << "Failed to get resource statistic";
```

Comment:
The error can cause build heartbeat failed, maybe `ERROR` level is better.

Reply:
fix